media/public/include/IMediaPipeline.h

```diff
@@ -1241,23 +1241,33 @@ class IMediaPipeline
     virtual bool renderFrame() = 0;
 
     /**
-     * @brief Set level and transition of audio attenuation.
-     *        Sets the current volume for the pipeline (0.0 silent -> 1.0 full volume)
+     * @brief Set the target volume level and transition duration with easing type.
+     *        If duration is 0 milliseconds, then volume is immediately set
```

Comment:
Maybe something like:
By default volume is set immediately if duration and type are not set.